guides/taxation/overview-of-taxation.md

```diff
@@ -0,0 +1,149 @@
+# Overview of taxation
+
+Solidus uses the `Spree::Adjustment` model to apply taxes to orders. This way,
+there can be multiple tax adjustments applied to each order.
+
+Using adjustments helps account for some of the complexities of tax, especially
+if a store sells internationally:
+
+- Orders may include products with different tax categories or rates.
+- Shipments may require special calculations if you are shipping to or from
+  locations where there are specific taxation rules for shipments.
+- Taxes may be included in a product's price or not in included in a product's
+  price depending on a country's taxation rules.
+
+<!-- TODO:
+  - Add links to the locations guide and specifically the [zones](#) article.
+  - Add links to the adjustments guide.
+-->
+
+Solidus uses the `Spree::TaxCategory` and `Spree::TaxRate` models to represent
+taxes for each line item in an order. See the sections below for more
+information about how these models work. 
+
+See the [Order taxation](#order-taxation) section for an overview of the order
+taxation process.
+
+## Tax categories
+
+Tax categories can be used to ensure particular products are taxed as required.
+For example, if your business is based in Minnesota, you need to charge tax on
+tech products but do not need to charge tax on clothing. You could set up two
+tax categories, **Clothing** and **Tech** which you would apply to products of
+either type.
+
+By default, new products do not have a set tax category. Administrators can set
+the tax category while creating the product or set it later while editing the
+product. 
+
+## Tax rates
+
+Tax rates define the amount of tax that should be charged on an item. You might
+configure different tax rates for different zones and tax categories, or even
+for specific dates. You could also create more complex tax rates with a custom
+tax calculator.
+
+In Solidus, a tax rate consists of at least four values:
+
+- The descriptive name for the tax rate. For example, "Minnesota" for a
+  Minnesota state tax rate.
+- The zone that the tax rate should apply to. <!-- TODO: Link to zones article.-->
+- The rate (in the form of a percentage of the price).
+- The "Included in price" boolean. This indicates whether the tax is included in
+  the price (for value-added taxes) or added to the price (U.S. taxes).
+
+Solidus calculates tax based on the matching tax rate(s) for the order's [tax
+address](#tax-addresses). 
+
+## Tax addresses
+
+If an order's `tax_address` falls within a specific zone, the tax rates that you
+have configured would apply for all the line items and shipments in that zone
+that have a matching tax category.
+
+By default, the `tax_address` used for orders is the customer's shipping
```

Comment:
I have not come across a scenario where the billing address is actually used for taxation. I believe we can actually get rid of the config option. Let's not spend too much time documenting it. 

One thing worth mentioning is that the `order.tax_address` can - through duck typing - be either an address or a `Spree::TaxLocation`, which is computed from the store's default cart tax country ISO. You can only trust the tax address has a country, all other fields might be empty or raise errors.

Reply:
Thanks for this.

What would be the use case for setting the `tax_address` to be a `TaxLocation`?